test/test_rsb_introspection.py

```diff
@@ -55,3 +59,26 @@ def test_construction_defaults(self):
         assert isinstance(info.machine_type, str)
         assert isinstance(info.software_type, str)
         assert isinstance(info.software_version, str)
+
+
+class TestIntrospectionEvents:
+
+    @pytest.mark.usefixture('rsb_config_socket')
+    def test_introspection_events(self):
+        # The listener must be able to deserialize introspection
```

Comment:
Could you please rewrap comments at 80 characters. This is used throughout the whole source code.

Reply:
Done.